src/bch/coin-selection.js

```diff
@@ -59,7 +59,7 @@ const findTarget = (targets, feePerByte, coins, changeAddress) => {
 
 // selectAll :: Number -> [Coin] -> String -> Selection
 const selectAll = (feePerByte, coins, outAddress) => {
-  let splitCoins = prepareForSplit(coins)
+  let splitCoins = coins; // prepareForSplit(coins)
```

Comment:
It would be nice to separate the `prepareForSplit` behavior from the coin selection.